x-pack/plugins/enterprise_search/public/applications/app_search/components/result_settings/result_settings_table/text_fields_header.tsx

```diff
@@ -14,42 +14,6 @@ import { i18n } from '@kbn/i18n';
 export const TextFieldsHeader: React.FC = () => {
   return (
     <>
-      <EuiTableHeader className="resultSettingsTable__columnLabels">
-        <EuiTableHeaderCell align="left" />
-        <EuiTableHeaderCell align="center" colSpan={2}>
-          {i18n.translate('xpack.enterpriseSearch.appSearch.engine.resultSettings.table.rawTitle', {
-            defaultMessage: 'Raw',
-          })}
-          <EuiIconTip
```

Comment:
I think you need to go back and remove the import for `EuiIconTip` in this file to get our linting to pass